tests/builders/schema/test_anyof.py

```diff
@@ -38,6 +38,16 @@
             ]
         )
     ),
+    (
+        {
+            "description": "Can be any value - string, number, boolean, array or object."
+        },
+        AnyOf(
+            type=DataType.ANY_OF,
+            description="Can be any value - string, number, boolean, array or object.",
+            schemas=[]
+        )
```

Comment:
would be nice to have one more case with some actual schema but w/o `type`, for instance some object or array with items

Reply:
Do you mean a test case along these lines:
```python
(
        {
            "description": "Array with implicit type.",
            "items": [
                {"type": "string"}
            ]
        },
        AnyOf(
            type=DataType.ANY_OF,
            description="Array with implicit type.",
            schemas=[]
        )
    )
 ```
Or do you mean that in the case with arrays/objects the type should be inferred? Reading this SO answer: https://stackoverflow.com/questions/47374980/schema-object-without-a-type-attribute-in-swagger-2-0/47390723#47390723 it seems that we maybe should not infer types.  What do you think?